scripts/scene/RoomBuilder.gd

```diff
@@ -181,6 +190,7 @@ func draw_rooms() -> void:
 	# Clear drafting layer
 	build_tile_map.clear_layer(drafting_layer)
 	build_tile_map.clear_layer(building_layer)
+	furniture_tile_map.clear_layer(hotspot_layer)
```

Comment:
I missed this in the last commit, but the hotspots were not being cleared on deleting a room.

Reply:
🙌 was about to make an issue but you caught it already